roadmap.md

```diff
@@ -1,192 +1,70 @@
+- [ ] Asynchronous commit.
+- [ ] Clustered Index.
+- [ ] Support cross-region deployment and geo-partition.
 
-* Support TiFlash storage engine
-* Support Optimizer Trace
-* Support multi-column statistics
-* Support TopN statistics for regular CM-Sketch
-* Improve the Plan Cache feature
-* Support self-adaptive SQL engine
-* Support SQL Tuning Advisor
-* Support SQL plan management
-* Transaction
-    + Pessimistic locking general availability (GA)
-    + Support an unlimited number of statements in a transaction
-    + Support 10 GB transactions
+## Improve system security
 
-#### Performance
+### Authentication
 
-* Improve CSV/data loading performance
-* Improve `Prepare` statement performance
-* Support index for generated columns
-* Optimize some operators of the SQL engine
-    + Improve performance of queries by using indexes to return to the table
-    + Split Index Join to Index Merge Join and Index Hash Join
-    + Radix Hash Join
-    + Index Merge
-    + Parallel Stream Aggregate
-    + Parallel Merge Sort
-    + Parallel Merge Join
-    + Full Vectorized Expression Evaluation
-* Indexes on expressions
-* Multi-index scan
-* Support external storage for Join, Aggregate, and Sort operators
-* Optimize the execution engine concurrency model
-* Support new Cascades Optimizer and Cascades Planner to increase the Optimizer searching space
+- [ ] Transport Layer Security(TLS) for TiFlash.
+- [ ] TLS for internal communication in the TiDB cluster.
+- [ ] SSH LDAP extension for TiUP.
 
-#### Usability
+### Transparent Data Encryption (TDE)
 
-* Improve the Optimizer Hint feature
-* Quickly restore database or table metadata and data
-* Dynamically modify configuration items
-* Automatically terminate idle connections
-* Improve support for DDL statements in MySQL 5.7
-* Refactor log content
-* Support `admin checksum from … to …` to verify data integrity
-* Support using standard SQL statements to query the DDL history
-* Support using standard SQL statements to manage Binlog
-* Support using standard SQL statements to manage the cluster
-* Integrate multiple Ctrl tools into one tool
+- [ ] Transparent Data Encryption (TDE) for TiFlash.
+- [ ] Transparent Data Encryption (TDE) for PD.
 
-#### High Availability
+### Mask
 
-* Support high service availability with TiDB Binlog
-* Support high data reliability with TiDB Binlog
+- [ ] Desensitize the TiDB general log.
 
-### TiKV Server
+## Cost-effectiveness
 
-#### Features
+- [ ] Optimize the performance and stability of TiDB running on AWS i3.xlarge/i3.2xlarge.
+- [ ] Optimize the performance and stability of TiDB running on non-NVMe SSD or on cloud disk (such as AWS EBS gp2).
+- [ ] Easier to discover performance issues and diagnose the causes.
 
-* Support up to 200+ nodes in a cluster
-* Fast full backup and restoration
-* Dynamically split and merge hot spot Regions
-* Fine-grained memory control
-* Raft
-    + Joint consensus
-    + Read-only replicas
+## New features
```

Comment:
how about collecting the features related to data migration to another section?

Reply:
The reason?